CHANGES/6774.removal

```diff
@@ -0,0 +1 @@
+Facilitate `pulp_workers` config
```

Comment:
I think we should have a more specific message.

Reply:
@mikedep333 can you suggest something

Reply:
```suggestion
Replaced `pulp_workers` dictionary variable with the `pulp_workers` integer variable. `pulp_workers` is now simply the number of workers.
```